src/core/psi_container.jl

```diff
@@ -92,6 +92,13 @@ function _make_jump_model!(psi_container::PSIContainer)
         parameters && PJ.enable_parameters(JuMPmodel)
         psi_container.JuMPmodel = JuMPmodel
     end
+    if !get_optimizer_log_print(settings)
+        JuMP.set_silent(psi_container.JuMPmodel)
+    elseif get_optimizer_log_print(settings)
+        JuMP.unset_silent(psi_container.JuMPmodel)
```

Comment:
This could be simpler.
```
    if get_optimizer_log_print(settings)
        JuMP.unset_silent(psi_container.JuMPmodel)
    else
        JuMP.set_silent(psi_container.JuMPmodel)
    end
```

Reply:
I wanted to do this just in case we change the setting later and be able to catch it. If that's ok with you I would ratter leave or change it by and @assert maybe?

Reply:
I don't I think I'm following you. The function returns either true or false. It's not really important...whatever you think is best.